app/models/jobsModel.js

```diff
@@ -1,3 +1,4 @@
+const { scheduledJobs } = require('node-schedule');
```

Comment:
This import doesn't seem to be necessary.

Reply:
Good Comments!